chore: add newspaper layout with CSS Grid and user stories

zairahira opened the pull request:
Checklist:

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

Closes #670

<!-- Feel free to add any additional description of changes below this line -->

Dario-DC commented on fullstack-cert/css-projects/newspaper-css-grid-lab/index.html:
```diff
@@ -0,0 +1,58 @@
+
+<body>
+    <div class="newspaper-layout">
+        <div class="title">The Daily Local News</div>
```
We need an `h1` for the title.

Dario-DC commented on fullstack-cert/css-projects/newspaper-css-grid-lab/index.html:
```diff
@@ -0,0 +1,58 @@
+        </div>
+
+        <div class="cover-image">
+            <img src="https://cdn.freecodecamp.org/curriculum/labs/alps.jpg" alt="Cover Image">
```
I would use another image here since this has been used in a previous lab.

Dario-DC reviewed (commented):
We should ask to include an `h2` and a `p` within `feature-article` since an `h3` and a `p` are required for the smaller articles. And now that I think about it, maybe `article` elements are more appropriate than `section` elements :thinking: After that I think we can merge this.

zairahira commented:
@Dario-DC,

I have added another story for the content of `.news-article`:

```md
4. The `.news-article` should include an `h2` element for the article title and a `p` element for the article content.
```

I have also updated the `section` elements to `article` both in the html and userstories.

Dario-DC commented on fullstack-cert/css-projects/newspaper-css-grid-lab/userstories.md:
```diff
@@ -4,27 +4,29 @@
+6. Each of the smaller articles should include an `h3` element for the article title and a `p` element for the article content.
 
-7. The `.newspaper-layout` should include a `figure` with the class `cover-image` to display an image that represents the newspaper's content.
+7. The `.newspaper-layout` should include a `article` element with the class `secondary-article` for an additional news section.
```
```suggestion
7. The `.newspaper-layout` should include an `article` element with the class `secondary-article` for an additional news section.
```

Dario-DC reviewed (approved):
LGTM :+1:
